apps/picsa-tools/manual-tool/src/app/components/stepsContainer/stepsContainer.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="mobileManualPage">
+  <div class="simpleSection">
+    <h3 class="ActivityHeading">Long before the season</h3>
+    <div class="ActivitiesContainer">
+      <div class="ActivityItem" *ngFor="let item of steps.LongBeforeSeason">
+        <div class="IconHolder">
+          <mat-icon class="iconStyle" svgIcon={{item.icon}}></mat-icon>
+         </div>
+          <span>{{ item.name }}</span>
+      </div>
+    </div>
+  </div>
+  <div class="simpleSection">
+    <h3 class="ActivityHeading">Just before the season</h3>
```

Comment:
nit(non-blocking)
It would probably be possible to do all this in one ngFor loop for all the steps combined, but not a big issue

Reply:
Surely meaning like the headings aswell